resolvme.gemspec

```diff
@@ -30,14 +30,14 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = "~> 2.6.5"
 
-  spec.add_development_dependency "bundler"
+  spec.add_development_dependency "bundler", "~> 2.0"
   spec.add_development_dependency "rake", "~> 12.0"
   spec.add_development_dependency "rspec", "~> 3.8"
-  spec.add_development_dependency "rdoc"
-  spec.add_development_dependency "yard"
-  spec.add_development_dependency "pry"
-  spec.add_development_dependency "pry-byebug"
-  spec.add_dependency "vault", "~> 0.12"
+  spec.add_development_dependency "rdoc", "~> 6.2"
+  spec.add_development_dependency "yard", ">= 0.9", "< 0.10"
+  spec.add_development_dependency "pry", ">= 0.12", "< 0.13"
+  spec.add_development_dependency "pry-byebug", "~> 3.8"
+  spec.add_dependency "vault", ">= 0.12", "< 0.13"
```

Comment:
This clashes with what's in devops-tools -> `gem "vault", "~> 0.13", "< 0.14"`; bundler will fail

Reply:
bugger, missed that.

Which one can/should we change? I'd suggest bringing this up to match devops-tools

Reply:
@michelesr anything breaking to resolvme between Vault 0.12 and 0.13?

Reply:
it will be best to remove the explicit dependency on vault from devops-tools

Reply:
Isn't the explicit reference needed because vault is a transitive dependency? Same with cfn-nag

Reply:
so... bundle install on master gemfile pointing to a github gem will install the gem and any non-development deps specified in the gemspec. It will not install deps listed in the gem's Gemfile. These must be added explicitly in the master gemfile.

cfn-nag is a dev dependency so it must be explicitly set. Vault is a runtime dependency so it's ok to remove it from devops-tools